posthog-node/src/feature-flags.ts

```diff
@@ -112,6 +114,29 @@ class FeatureFlagsPoller {
     return response
   }
 
+  async getFeatureFlagPayload(key: string, matchValue: string | boolean): Promise<JsonType | undefined> {
```

Comment:
open question remains, to pass `matchValue` in or not. I'm still on the side of not passing in, as your argument around "makes for zero user erroneous input" stands strong in server-side libs too.

Reply:
This line you're referring to is the poller so it requires matchValue but in terms of the interface, is it detrimental to leave the implementation as is? We can document the usage as if you don't pass it in, but also add documentation if it seems it's a legit use case

Reply:
Ah, this isn't exposed to the client 🤦 - yeah it's perfectly fine then!